valhalla/baldr/traffictile.h

```diff
@@ -149,8 +151,11 @@ class TrafficTile {
   }
 
   const volatile TrafficSpeed& trafficspeed(const uint32_t directed_edge_offset) const {
-    if (header == nullptr)
+    if (header == nullptr) {
+      return INVALID_SPEED;
+    } else if (header->traffic_tile_version != TRAFFIC_TILE_VERSION) {
```

Comment:
Why not have one if block and use an ||

Reply:
Logged different messages while debugging, I just pushed a commit that collaps them :+1: